app/models/default_pet_task.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: default_pet_tasks
+#
+#  id              :bigint           not null, primary key
```

Comment:
Do we want to enforce order on these? If so, https://github.com/brendon/acts_as_list could be a nice addition

Reply:
Oh yes good catch I forgot about that requirement.

Reply:
@edwinthinks so is the use of acts_as_list primarily for implement drag and drop ordering of tasks? Are we thinking drag and drop is MVP?

If we just want to enforce order, are you thinking on the default task we would have a `position` attribute that is used to set the same attribute on the Task for ordering tasks in the UI? That makes sense. But, what about for newly created tasks (not default)? We would have to set the `position` on the back end e.g., get the last Task's `position` and increment it by one. But then we'd have to handle simultaneous writes and race conditions.

I am thinking about this task list for MVP...in the simplest way possible. Feel free to shoot this down...

Why don't we just list uncompleted tasks by the created_at date? Then all default tasks will be above newly created tasks. Then list completed tasks beneath all of the uncompleted tasks. It's a simple way to keep the uncompleted stuff at the top. Recurring tasks will be added to this list, also. The way I see it, over time, the older default tasks should be completed, so will move to the bottom, and newly created, uncompleted tasks will remain at the top. This will work well for recurring tasks also.

For due dates, that's easily handled in MVP with a due date attribute, we display due date next to task name, and we just check if today is past this date, and if so, display a red overdue or the likes.

Reply:
> We would have to set the position on the back end e.g., get the last Task's position and increment it by one. But then we'd have to handle simultaneous writes and race conditions.

Good point. Though I don't think it is much of a case if we aren't dealing with high concurrency. There is a another gem for that which is similiar that handles that.

> I am thinking about this task list for MVP...in the simplest way possible. Feel free to shoot this down...

Thanks for sharing your thoughts! I agree let's keep the MVP simple. No need to add this if its going to add extra burden to releasing.